Backpack/NavigationBar/Classes/BPKNavigationBar.m

```diff
@@ -366,7 +366,7 @@ - (void)updateAppearance {
 }
 
 - (UIColor *)borderViewBackgroundColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.surfaceHighlightColor darkVariant:BPKColor.textSecondaryColor];
+    return [BPKColor dynamicColorWithLightVariant:BPKColor.surfaceHighlightColor darkVariant:BPKColor.lineDarkColor];
 }
```

Comment:
This can be changed to use BPKColor.lineColor (Figma uses LineColor both dark and light mode)

Reply:
Good suggestion. I am updating it now